packages/bot/src/stores/journal-store.js

```diff
@@ -0,0 +1,59 @@
+
+import { observable, action } from 'mobx';
+import { formatDate }         from 'deriv-shared/utils/date';
+import { observer }           from '../utils/observer';
+
+export default class JournalStore {
+    constructor(rootstore) {
+        this.rootstore = rootstore;
+
+        observer.register('ui.log.success', this.onLogSuccess);
```

Comment:
Make sure to create a disposer function for these listeners (for when the component unmounts).

```JavaScript
observer.unregister('ui.log.success', this.onLogSuccess);
```